.github/workflows/reusable_kernel_tests.yaml

```diff
@@ -26,6 +24,15 @@ jobs:
         architecture: [X64, ARM64]
     runs-on: [ "self-hosted", "linux", "${{matrix.architecture}}" ]    
     steps:
+      - name: Wait for any other running kernel_tests workflow to finish
```

Comment:
NOTE: i am not sure this is going to work; basically, it is needed because we can't define a `concurrency` in the reusable workflow, BUT we need to avoid concurrent run of the kernel-tests.
This way, we should be able to wait for other kernel-tests to finish before proceeding.

Reply:
Put this in `wip` for further refinements.